# contracts/ERC20Confidential/ERC20ConfidentialUpgradeable.sol  ->  contracts/ERC20Confidential/ERC20ConfidentialUpgradeable.fsol
# upstream 5138cb8 .. ported 86f1b7c

diff --git a/contracts/ERC20Confidential/ERC20ConfidentialUpgradeable.sol b/contracts/ERC20Confidential/ERC20ConfidentialUpgradeable.fsol
index 1f4691c..dd069a0 100644
--- a/contracts/ERC20Confidential/ERC20ConfidentialUpgradeable.sol
+++ b/contracts/ERC20Confidential/ERC20ConfidentialUpgradeable.fsol
@@ -7,10 +7,10 @@ import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/I
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { IERC165 } from "@openzeppelin/contracts/interfaces/IERC165.sol";
 import { ERC165Upgradeable } from "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
-import { IFHERC20, IERC7984 } from "../interfaces/IFHERC20.sol";
-import { IERC20Confidential } from "../interfaces/IERC20Confidential.sol";
-import { ERC20ConfidentialIndicator } from "./ERC20ConfidentialIndicator.sol";
-import { ERC20ConfidentialCoreUpgradeable } from "./ERC20ConfidentialCoreUpgradeable.sol";
+import { IFHERC20, IERC7984 } from "../interfaces/IFHERC20.fsol";
+import { IERC20Confidential } from "../interfaces/IERC20Confidential.fsol";
+import { ERC20ConfidentialIndicator } from "./ERC20ConfidentialIndicator.fsol";
+import { ERC20ConfidentialCoreUpgradeable } from "./ERC20ConfidentialCoreUpgradeable.fsol";
 
 /**
  * @title ERC20ConfidentialUpgradeable
